pkg/rt/manager.go

```diff
@@ -102,47 +124,59 @@ func (m *Manager) Refresh() {
 
 // Active returns the set of service IDs currently being managed.
 // Mostly useful for tests.
-func (m *Manager) Active() (serviceIDs []string) {
+func (m *Manager) Active() []string {
+	serviceIDs := []string{}
```

Comment:
This is a small thing, but we know what the size of `serviceIDs` is going to be. We should call `m.managedKeysWithLock()` and use the length of the return value as a parameter to `make()` to create `serviceIDs`.